examples/inventory_example.rb

```diff
@@ -26,7 +26,7 @@ def print_object(base, caption, indent = 0, recurse = true)
         puts "#{indentation}Ignoring #{rel} relationship due to ovirt error: #{err}"
       rescue RestClient::InternalServerError => err
         puts_caption(new_caption, indentation)
-        puts "#{indentation}Ignoring #{rel} relationship due to rest client error: #{err}"
+        puts "#{indentation}Ignoring #{rel} relationship due to rest client error: #{err.response}"
```

Comment:
I don't think so, it's just an example of how to use the inventory part of the code.